graspy/inference/latent_distribution_test.py

```diff
@@ -14,121 +14,158 @@
 
 import numpy as np
 
-from ..embed import AdjacencySpectralEmbed, select_dimension
+from ..embed import select_dimension, AdjacencySpectralEmbed
 from ..utils import import_graph, is_symmetric
 from .base import BaseInference
+from sklearn.metrics import pairwise_distances
+from sklearn.metrics.pairwise import PAIRED_DISTANCES
+from hyppo.ksample import KSample
+
+_VALID_METRICS = list(PAIRED_DISTANCES.keys())
+_VALID_METRICS.append("gaussian")  # we have a gaussian kernel implemented too
+_VALID_TESTS = ["cca", "dcorr", "hhg", "rv", "hsic", "mgc"]
 
 
 class LatentDistributionTest(BaseInference):
     """
-    Two-sample hypothesis test for the problem of determining whether two random 
+    Two-sample hypothesis test for the problem of determining whether two random
     dot product graphs have the same distributions of latent positions.
-    
+
     This test can operate on two graphs where there is no known matching between
-    the vertices of the two graphs, or even when the number of vertices is different. 
+    the vertices of the two graphs.
     Currently, testing is only supported for undirected graphs.
 
     Read more in the :ref:`tutorials <inference_tutorials>`
 
     Parameters
     ----------
+    test : str
+        Backend hypothesis test to use, one of ["cca", "dcorr", "hhg", "rv", "hsic", "mgc"].
+        These tests are typically used for independence testing, but here they are used 
+        for a two-sample hypothesis test on the latent positions of two graphs. See 
+        :class:`hyppo.ksample.KSample` for more information. 
+
+    metric : str or function, (default="euclidean")
+        Distance metric to use, either a callable or a valid string.
+        The callable should behave similarly to :func:`sklearn.metrics.pairwise_distances`,
+        if a string should be one of the keys in `sklearn.metrics.pairwise.PAIRED_DISTANCES`
+        or "gaussian" which will use a Gaussian kernel on Euclidean distances.
```

Comment:
reminder to specify that "gaussian" is a kernel with an adaptively selected bandwidth

Reply:
fixed.